app/settings.py

```diff
@@ -43,51 +23,44 @@ def sentry_filter_transactions(event: dict[str, Any], hint: dict[str, Any]) -> d
-    twitter_link: str = config_env.get("TWITTER_LINK", "www.twitter.com")
-    link_link: str = config_env.get("LINK_LINK", "http://localhost:3000")
-    test_mail_receiver: str = config_env.get("TEST_MAIL_RECEIVER", "credere@noreply.open-contracting.org")
-    colombia_secop_app_token: str = config_env.get("COLOMBIA_SECOP_APP_TOKEN", "")
-    hash_key: str = config_env.get("HASH_KEY", "")
-    application_expiration_days: int = config_env.get("APPLICATION_EXPIRATION_DAYS", 7)
-    secop_pagination_limit: int = config_env.get("SECOP_PAGINATION_LIMIT", 5)
-    secop_default_days_from_ultima_actualizacion: int = config_env.get(
-        "SECOP_DEFAULT_DAYS_FROM_ULTIMA_ACTUALIZACION", 365
-    )
-    days_to_erase_borrower_data: int = config_env.get("DAYS_TO_ERASE_BORROWERS_DATA", 7)
-    days_to_change_to_lapsed: int = config_env.get("DAYS_TO_CHANGE_TO_LAPSED", 14)
-    ocp_email_group: str = config_env.get("OCP_EMAIL_GROUP", "credere@noreply.open-contracting.org")
-    max_file_size_mb: int = config_env.get("MAX_FILE_SIZE_MB", 5)
-    reminder_days_before_expiration: int = config_env.get("REMINDER_DAYS_BEFORE_EXPIRATION", 3)
-    progress_to_remind_started_applications: float = config_env.get("PROGRESS_TO_REMIND_STARTED_APPLICATIONS", 0.7)
-    environment: str = config_env.get("ENVIRONMENT", "development")
-    transifex_token: str = config_env.get("TRANSIFEX_TOKEN", "")
-    transifex_secret: str = config_env.get("TRANSIFEX_SECRET", "")
-
-    class Config:
-        env_file = ".env"
+    version: str = VERSION
+    log_level: int | str = logging.INFO
+    aws_region: str = "us-west-2"
+    cognito_pool_id: str = ""
+    aws_access_key: str = ""
+    aws_client_secret: str = ""
+    email_sender_address: str = "credere@noreply.open-contracting.org"
+    cognito_client_id: str = ""
+    cognito_client_secret: str = ""
+    database_url: str = ""
+    test_database_url: str = ""
+    frontend_url: str = "http://localhost:3000"
+    sentry_dsn: str = ""
+    images_base_url: str = "https://credere.open-contracting.org/images"
+    email_template_lang: str = "es"
+    facebook_link: str = "www.facebook.com"
+    twitter_link: str = "www.twitter.com"
+    link_link: str = "http://localhost:3000"
+    test_mail_receiver: str = "credere@noreply.open-contracting.org"
+    colombia_secop_app_token: str = ""
+    hash_key: str = ""
+    application_expiration_days: int = 7
+    secop_pagination_limit: int = 5
+    secop_default_days_from_ultima_actualizacion: int = 365
+    days_to_erase_borrower_data: int = 7
+    days_to_change_to_lapsed: int = 14
+    ocp_email_group: str = "credere@noreply.open-contracting.org"
+    max_file_size_mb: int = 5
+    reminder_days_before_expiration: int = 3
+    progress_to_remind_started_applications: float = 0.7
+    environment: str = "development"
+    transifex_token: str = ""
+    transifex_secret: str = ""
+    model_config = SettingsConfigDict(env_file=".env", extra="allow")
```

Comment:
Just curious - what are these extras that we want to allow? Should we make them explicit settings?

Reply:
Good catch, the issue was with `days_to_erase_borrowers_data` which was named `days_to_erase_borrower_data` but days_to_erase_borrowers_data in the env file. Now we don't need `extra=allow` anymore